src/sqlitestorage.cpp

```diff
@@ -948,7 +899,7 @@ int SqliteStorage::loadIncidences(bool hasDate, int limit, QDateTime *last)
     SL3_prepare_v2(d->mDatabase, query1, qsize1, &stmt1, NULL);
     SL3_bind_int64(stmt1, index, secsStart);
 
-    count = d->loadIncidences(stmt1, limit, last, hasDate);
+    count = d->loadIncidences(stmt1, true, limit, last, hasDate);
```

Comment:
To note here I needed to check what's "loadIncidences()" again as the name is quite generic. Based on the documentation description I'm not immediately sure what's it about as it's talking creation or start/due.

Oh well, maybe this and all here would be subject to be killed :)